src/server/host/SubscriptionHost/types/path-fn.ts

```diff
@@ -0,0 +1,3 @@
+export interface PathFn<T, V> {
```

Comment:
Can this be declared locale to `action-field.tsx`?

Reply:
Sure